transaction/transaction_test.go

```diff
@@ -132,8 +132,10 @@ func TestMakeAssetCreateTxn(t *testing.T) {
 	const clawback = addr
 	const unitName = "tst"
 	const assetName = "testcoin"
+	const testURL = "website"
+	const metadataHash = "fACPO4nRgO55j1ndAK3W6Sgc4APkcyFh"
```

Comment:
is that a real 32byte hash?
While the code should not reject a shorter than 32 bytes string (If someone decides to make another use of the field, I'm ok with it), it should reject longer ones. 
Also, for the sake of completeness, if this is not a 32 byte string, I'd use a real one.